pkg/asset/machines/master.go

```diff
@@ -417,6 +421,19 @@ func (m *Master) Generate(dependencies asset.Parents) error {
 			return errors.Wrap(err, "failed to create master machine objects")
 		}
 		vsphere.ConfigMasters(machines, clusterID.InfraID)
+	case powervstypes.Name:
+		mpool := defaultPowerVSMachinePoolPlatform()
+		mpool.Set(ic.Platform.PowerVS.DefaultMachinePlatform)
+		mpool.Set(pool.Platform.PowerVS)
+		// Only the service instance is guaranteed to exist and be passed via the install config
+		// The other two, we should standardize a name including the cluster id. At this point, all
+		// we have are names.
```

Comment:
Is this the right place for this comment? A similar comment is in `Machines` function. I don't see a context for service instance or "the other two" here.

Reply:
ah no, it isn't. the _other two_ are `image` and `network` in `Machines()` and `Machinesets()`. ill delete this